### PR TITLE
Standard button

### PR DESCRIPTION
**Pull request by robeeeert**

### CODE REVIEW

File: src/LoginButton.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+  }
+
+  render () {
+    const { type, flavour, children, withIcon, ...restProps } = this.props
+
+    return (
+      <button
+        type={type}
+        className={`
```
Comment (dan-lee) [inline src/LoginButton.jsx]: Better rewrite that as `{...css(styles.button, this.props.disabled && styles.disabled, ... }`

File: src/LoginButton.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    if (this.props.onClick) {
+      this.props.onClick(e)
+    }
+  }
+
+  render () {
+    const { type, flavour, children, withIcon, ...restProps } = this.props
+
+    return (
+      <button
```
Comment (dan-lee) [inline src/LoginButton.jsx]: Could you use your fresh & fancy `Button` component here? 🐄

File: stories/Button.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+export default class ButtonStory extends React.Component {
+  constructor () {
+    super()
+
+    this.state = {
+      primary: true,
+      secondary: false
+    }
+
+    this.toggle = ::this.toggle
```
Comment (dan-lee) [inline stories/Button.js]: 👍

**Comment by robeeeert** [comment]
[Put general button into app-ui (c1)](https://trello.com/c/VLajYSNJ/2219-put-general-button-into-app-ui-c1)

**Comment by robeeeert** [comment]
[Put login button into app-ui (c1)](https://trello.com/c/X8678y0s/2218-put-login-button-into-app-ui-c1)

**Comment by robeeeert** [comment]
[Put titlebar items into app-ui (c2)](https://trello.com/c/FEKT8WRe/2239-put-titlebar-items-into-app-ui-c2)

**Comment by dan-lee** [comment]
Go for it Roberton! 🤖 🐄

**Pull request merged**